uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -487,14 +477,12 @@ public function testSaveFromEntitlementCalculationCanSaveOverriddenValuesGreater
     $this->assertEquals($contact['id'], $periodEntitlement->contact_id);
     $this->assertEquals(1, $periodEntitlement->overridden);
     $this->assertEquals($overriddenEntitlement, $periodEntitlement->getEntitlement());
+
+    $this->unregisterCurrentLoggedInContactFromSession();
   }
 
   public function testSaveFromEntitlementCalculationCanSaveOverriddenValuesLessThanTheProposedEntitlement() {
-    // This mocks the logged in user so we can test
-    // the LeavePeriodEntitlement creation with a comment
-    global $user;
-    $user = new stdClass();
-    $user->uid = 1;
+    $this->registerCurrentLoggedInContactInSession(1);
```

Comment:
We can remove this just as I suggested above. Then we have a separate test to verify that entitlements are properly saved with comments.